### DISCUSSION TITLE
What makes a successful project at the GSF

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/oc#19] [General]
![image](https://user-images.githubusercontent.com/897523/222196684-288d8885-6b57-43a4-b121-db04237f0b0b.png)

Been thinking how to explain this. I'm very used to working in corporate environments where you set goals and people just get on with doing work. If the team isn't interested in doing the work, it doesn't matter, it's their job to implement a decision.

The foundation is different because we're driven through volunteers, you can't force anyone to do anything, nothing happens in the foundation unless at least one person is interested enough in the outcome to drive it.

Sometimes I see frustrations in people or just ideas which assumes a model closer to a traditional corp. structure were we can just decide things and they happen.

In the Foundation it's 100% about finding people who want to work on **something**, seeing if that **thing** aligns with the goals of the Foundation, deciding if it's actually possible, and then pouring petrol onto that fire.

Every project starts with one person and an idea, the successful projects manage to attract other people who like the work and want to get involved.

So anytime I see anyone proposing a project that falls in the green Venn diagram above I do everything I can to help them succeed!

I'm not sure who this message is for, but I just realized I could explain this tough concept in my mind with a Venn diagram so I went for it :)

**Comment by greenhsu123** [comment]
I seconded 100% (I wonder if this makes sense!). However, many volunteers are okay with being told what to do if they know this project will succeed, which we can see happening with many well-known/mature-enough open-source projects, like the otel-collector. They know it's "the" industry standard, so they wouldn't mind doing less exciting work. 

While the foundation matures and our projects/work gets more visibility, have we considered putting a timeframe on when someone can propose a project? The way I see it now is that we have too many projects and too few volunteers. Can the SC or OC devise a list of yearly priority projects and encourage people to focus on those? This strategy may lead to quicker and bigger wins... I think ... This would make us a bit more corporate, though :(

**Reply by jawache** [reply]
I see you're point and I agree that within one project people are more willing to just take on whatever task related to that project. I don't think that works between projects though? e.g. if we cancelled the meetup program I don't think the meetup organizers would put their effort into say an opensource project, they just wouldn't do anything IMO. So I don't think volunteers are that fungible between projects. 

The feedback I tend to get (I try to do a lot of recruitment behind the scenes) is either they didn't know about the project (or enough details about the project to know it's something they want to work on) or it's not clear at all how to start engaging on a project (other than turning up to meetings which is challenging for a lot of people). So there is a desire amongst volunteers but we're somehow not doing a great job of selling our projects to volunteers or on-boarding/on-ramping them to projects or there isn't a project that speaks to their skills/wants/desires.

That's one of the reasons for the digest emails, get more people engaged on projects by surfacing the most popular issues to them every week in their inbox. We've got some more phases to that whole project will share it in the next OC call.

RE: A time limit, I do like the idea of a time limit on incubation projects, they should read some (very low) bar within 6 months or we call it and close the project perhaps? We've had to make the call on a few projects recently.

RE: Raising the visibility of projects, I think the OC can flag certain projects as more important, we'd need a process for that and defo by consensus. 

One thing I was thinking of and have been meaning to raise in the Community WG discussions forum, is the idea of Green Software Experts. We've been talking of like an ambassador style program in the GSF since before the launch. So like the Microsoft MVP badge, or Google Cloud Experts. To earn that badge you have to work hard, demonstrate lots of Green Software expertise which should include contributing to our projects. That could be a way to have a pool of people who have a reason to engage with our projects outside of simple interest/desire.
